applications/new_schema.py

```diff
@@ -71,6 +76,34 @@ def get_node(cls, info, id):
         return super().get_node(info, id)
 
 
+class UpdateBerthApplication(graphene.ClientIDMutation):
+    class Input:
+        id = graphene.ID(required=True)
+        customer_id = graphene.ID(required=True)
+
+    berth_application = graphene.Field(BerthApplicationNode)
+
+    @classmethod
+    @login_required
+    @superuser_required
+    @transaction.atomic
+    def mutate_and_get_payload(cls, root, info, **kwargs):
```

Comment:
the `kwargs` here could be renamed to `input` for more clarity, [ref](https://github.com/City-of-Helsinki/open-city-profile/blob/develop/profiles/schema.py#L361)

Reply:
Agree, `**input` makes more sense, but we'd also have to refactor all the mutations in [resources](https://github.com/City-of-Helsinki/berth-reservations/blob/da62f2ea6868a2f9eda35c66166d84c68b5fbec8/resources/schema.py)

Reply:
yes, let's create a ticket for the refactor

Reply:
ok, I'll push a refactor commit